scripts/integration_test.sh

```diff
@@ -22,6 +22,14 @@ ls -lh data
 # Removes it to ensure the commit retrieval work as expected
 rm data/commit*
 
+# Then generate a test dataset of fixed inline comments
+bugbug-fixed-comments --patch-threshold 150 --diff-length-threshold 1000
```

Comment:
We may rely on the default value from the CLI for the diff length.

Reply:
Done here: 0e2cc4fe3fd318040c2174981498609b08737517